sdk/python/feast/dqm/profilers/ge_profiler.py

```diff
@@ -0,0 +1,126 @@
+import json
```

Comment:
Do we need to folder `/dqm/` ? Or would the following `sdk/python/feast/profilers/ge_profiler.py` be enough ?

Reply:
It's doesn't sound very clear what is `feast.profilers` module for. So I thought `feast.dqm` would be a useful namespace.